rl_loop/train_and_validate.py

```diff
@@ -53,10 +57,14 @@ def train():
     new_model_name = shipname.generate(new_model_num)
     print("New model will be {}".format(new_model_name))
     save_file = os.path.join(fsdb.models_dir(), new_model_name)
+    #freeze_tpu requires the work_dir flag to be set
+    if not FLAGS.work_dir:
+        FLAGS.work_dir = fsdb.working_dir()
 
-    cmd = ['python3', 'train.py', training_file,
+    cmd = ['python3', 'train.py', '__unused_file__',
```

Comment:
__unused_file__?

Reply:
train.py can also take a list of files for when you don't use bigtables.  this should probably be refactored.  i'm gonna leave a TODO